client/src/components/Ticketing/GroupeToggle.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import ToggleButton from '@mui/material/ToggleButton';
+import ToggleButtonGroup from '@mui/material/ToggleButtonGroup';
+
+const GroupToggle = () => {
+  const [filterAs, setFilter] = React.useState('Active');
```

Comment:
For consistency sake, the second item in the use state should be named the same except with a prepended `set`. So this can either be `[filterAs, setFilterAs]` or `[filter, setFilter]`

Reply:
I also imagine that this should exactly match the value fields below, so `'active'`. This way `active` will be selected by default when the page loads

Reply:
This field would also be a good prop to add to this component. Something like `defaultValue` that could only be `active`, `inactive` or `all` and then gets thrown into the `useState` as the baseline

Reply:
added/fixed